hackathon_site/hardware/serializers.py

```diff
@@ -89,6 +91,18 @@ def get_team_code(obj: Order):
         return obj.team.team_code
 
 
+class IncidentCreateSerializer(serializers.ModelSerializer):
```

Comment:
`IncidentCreateSerializer` is defined twice?

I think you're fine with removing this one, the way you have it above with `IncidentListSerializer` subclassing `IncidentCreateSerializer` should work (and is a clever way of just adding the nested serializers for specific fields, we should keep that in mind).